inspirehep/dojson/hep/fields/bd01x09x.py

```diff
@@ -189,7 +189,7 @@ def get_value(val):
         return {
             'a': val.get('value'),
             '9': val.get('source'),
-            '2': val.get('type') or "DOI"
+            '2': val.get('schema') or "DOI"
```

Comment:
yes, DOIs should **never** be present in `persisten_identifiers`, they have their own place, `dois`.